assets/src/design-system/testUtils/sampleData.js

```diff
@@ -0,0 +1,178 @@
+    value: 'label-item-two',
+  },
+  {
+    label: 'label item three',
+    value: 'label-item-three',
+  },
+  {
+    label: 'label item four (disabled)',
+    value: 'label-item-four',
+    disabled: true,
+  },
+  {
+    label: 'label item five (value is a number)',
+    value: 5,
+  },
+  {
+    label: 'label item six (value is a boolean)',
+    value: true,
+  },
+  {
+    label: 'drafts',
+    value: 'drafts',
+  },
+  {
+    label: 'cats',
+    value: 'cats',
+  },
+  {
+    label: 'dogs',
+    value: 'dogs',
+  },
+  {
+    label: 'parakeets',
+    value: 'parakeets',
+  },
+  {
+    label: 'lemurs',
+    value: 'lemurs',
+  },
+  {
+    label: 'ocelots',
+    value: 'ocelots',
+  },
+];
+
+export const reallyLongOptions = [
+  {
+    label: 'mad mad mad mad world',
+    value: 'tears for fears',
+  },
+  {
+    label: 'bring on the dancing horses',
+    value: 'echo',
+  },
+  {
+    label: 'one 2 three four, uno dos tres rumba',
+    value: 'pitbull',
+  },
+];
+
+export const effectChooserOptions = [
+  {
+    value: 'none',
+    label: 'none',
+    width: 'full',
+  },
+  {
+    value: 'drop in',
+    label: 'drop',
+    width: 'full',
+  },
+  {
+    value: 'fly in left-to-right',
+    label: 'fly in',
+    width: 'half',
+  },
+  {
+    value: 'fly in top-to-bottom',
+    label: 'fly in',
+    width: 'half',
+  },
+  {
+    value: 'fly in right-to-left',
+    label: 'fly in',
+    width: 'half',
+  },
+  {
+    value: 'fly in bottom-to-top',
+    label: 'fly in',
+    width: 'half',
+  },
+  {
+    value: 'pulse',
+    label: 'pulse',
+    width: 'full',
+  },
+  {
+    value: 'rotate in left-to-right',
+    label: 'rotate in',
+    width: 'half',
+  },
+  {
+    value: 'rotate in right-to-left',
+    label: 'rotate in',
+    width: 'half',
+  },
+  {
+    value: 'twirl',
+    label: 'twirl',
+    width: 'full',
+  },
+];
+
+export const nestedDropDownOptions = [
+  {
+    label: 'aliens',
+    options: [
+      { value: 'alien-1', label: 'ET' },
+      { value: 'alien-2', label: 'Stitch' },
+      { value: 'alien-3', label: 'Groot' },
+      { value: 'alien-4', label: 'The Worm Guys' },
+      { value: 'alien-5', label: "Na'vi" },
+      { value: 'alien-6', label: 'Arachnids' },
+      { value: 'alien-7', label: 'The Predator' },
+      { value: 'alien-8', label: 'Xenomorph' },
+    ],
+  },
+  {
+    label: 'dragons',
+    options: [
+      { value: 'dragon-1', label: 'Smaug' },
+      { value: 'dragon-2', label: 'Mushu' },
+      { value: 'dragon-3', label: 'Toothless' },
+      { value: 'dragon-4', label: 'Falkor' },
+      { value: 'dragon-5', label: 'Drogon' },
+      { value: 'dragon-6', label: 'Kalessin' },
+    ],
+  },
+  {
+    label: 'dogs',
+    options: [
+      { value: 'dog-1', label: 'Snoopy' },
+      { value: 'dog-2', label: 'Scooby' },
+    ],
+  },
+  {
+    label: 'tricky content',
+    options: [
+      { value: 0, label: '0 as a number' },
+      { value: false, label: 'false as a boolean' },
```

Comment:
<img width="472" alt="Screen Shot 2021-01-04 at 9 00 19 AM" src="https://user-images.githubusercontent.com/1738349/103548394-5afe1400-4e6b-11eb-92f9-d0209fcedf40.png">

The problem with these are you cannot select falsy values. Now this is just a story so I'm not sure we need to address this, but ideally we'd never have an option with a falsy value because it gets ignored on select.

Reply:
I can drop the example, I'd initially done this to catch edge cases as developing the dropDown. You're right that we shouldn't have a false value in the dropDown though. 

I think I'll remove them from storybook, but set up a new test in the DropDown with this bad data to make sure it's not getting selected.

Reply:
Made more sense to just test in the utils where sanitizing is happening, moved the "bad options" to their own example.